blockchain/contracts/HelloWorld.sol

```diff
@@ -29,16 +37,20 @@ contract HelloWorld {
         nftOwner = payable(0xC77787e364E3420c0609a249F18De47430900f0C);
     }
 
-    function createDrop(uint256 _dropTime) public {
-        uint256 i;
-        // I choose "3" as a value for testing reasons
-        for (i = 0; i < 3; i++) {
+    function createDrop(uint256 _dropTime, uint256 _numberOfNFTS) public {
+        for (uint256 i = 0; i < _numberOfNFTS; i++) {
             mockNFT(_dropTime, i + 1);
         }
+        // We need to let users buy 1 NFT instead of 5% if there are less than 20
+        if (_numberOfNFTS < 20) {
+            fivePercent = 1;
```

Comment:
Is there a way to provide a name which is more independent from the hardcoded implementation?
Like
```
purchaseLimit = (_numberOfNFTs * percentLimit) / 100;
nftPurchaseLimit = max(1, purchaseLimit);
```

I don't know however, if such a `max` function exists (for convenience).